chainerx_cc/chainerx/cuda/numeric.cuh

```diff
@@ -48,6 +48,10 @@ __device__ inline T Sign(T x) {
     return IsNan(x) ? x : static_cast<T>(static_cast<int>(T{0} < x) - static_cast<int>(x < T{0}));
 }
 
+template <>
+__device__ inline uint8_t Sign(uint8_t x) {
+    return x > 0;
```

Comment:
Let's avoid implicit conversion from `bool` to `uint8_t`.

Reply:
Fixed.